test/evergreen.yml

```diff
@@ -1501,6 +1501,17 @@ tasks:
         vars:
           checkpoint_args: -t c -n 1000000 -k 5000000 -C cache_size=100MB
 
+  - name: test-unit-ppc
+    commands:
+      - func: "get project"
```

Comment:
The current convention is we have a compile task and the test tasks will depend on that compile task. As you can see, it is clunky as we still need to rebuild in the test tasks because of something silly in the autotools. There is a ticket to work on removing the rebuild in the future. But here I think we need to adhere to the convention to decouple the build task and the test.

Reply:
Created a new compile task "compile-pcc" which depends on "test-unit-ppc".

Reply:
here we should use fetch artifacts